### PR TITLE
New Customer Credit Dev. Guide

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [X] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

PR created to publish the new Dev. Guide - Managing a Customer Credit invoice (EDU-11818).

### CODE REVIEW

File: docs/guides/Integration-Guides/customer-credit-integration-guide/managing-a-customer-credit-invoice.md [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+
+> ℹ️ You can also check some details of a Customer Credit invoice details using the VTEX Admin. To do that, go to __Apps > Customer Credit > Invoices__, and click on the desired invoice.
+
+## Checking invoice details on a Customer Credit account
+
+To check details of all invoices issued on a customer credit account, you can use the [Retrieve invoices by customer credit account ID](https://developers.vtex.com/docs/api-reference/customer-credit-api#get-/api/creditcontrol/accounts/-creditAccountId-/invoices) endpoint. In this request, you must send the account identification (`creditAccountId`) as a path parameter.
+
+See an example below using the account `111111111_CPF`:
+
+__GET - Retrieve invoices by customer credit account ID__
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply docs/guides/Integration-Guides/customer-credit-integration-guide/managing-a-customer-credit-invoice.md:71]: In this case it is really bold, two underlined characters.

File: docs/guides/Integration-Guides/customer-credit-integration-guide/managing-a-customer-credit-invoice.md [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+
+`https://{accountName}.{environment}.com.br/api/creditcontrol/accounts/11111111111_CPF/invoices/`
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply docs/guides/Integration-Guides/customer-credit-integration-guide/managing-a-customer-credit-invoice.md:73]: Great. I will add the code blocks to the link references.

File: docs/guides/Integration-Guides/customer-credit-integration-guide/managing-a-customer-credit-invoice.md [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+    "numberOfInstallments": 1,
+    "creditAccountId": "11111111111_CPF"
+}
+```
+
+> ℹ️ You can also check some details of a Customer Credit invoice details using the VTEX Admin. To do that, go to __Apps > Customer Credit > Invoices__, and click on the desired invoice.
+
+## Checking invoice details on a Customer Credit account
+
+To check details of all invoices issued on a customer credit account, you can use the [Retrieve invoices by customer credit account ID](https://developers.vtex.com/docs/api-reference/customer-credit-api#get-/api/creditcontrol/accounts/-creditAccountId-/invoices) endpoint. In this request, you must send the account identification (`creditAccountId`) as a path parameter.
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply docs/guides/Integration-Guides/customer-credit-integration-guide/managing-a-customer-credit-invoice.md:67]: Great. I will check it.

**Review by PedroAntunesCosta — APPROVED** [review]
LGTM

**Pull request merged**